packages/react-ui-core/src/Ratings/LinearGradient.js

```diff
@@ -3,9 +3,10 @@ import PropTypes from 'prop-types'
 
 export default class LinearGradient extends Component {
   static propTypes = {
-    id: PropTypes.string,
-    color: PropTypes.string,
-    width: PropTypes.string,
+    id: PropTypes.string.isRequired,
```

Comment:
I don't mind making these required, but I almost wonder if `id` should be used in `context` instead.